### PR TITLE
fixed #155 Add a config file for yamllint

### LINKED ISSUE #155: [tools] add a config file for yamllint

### PR DESCRIPTION
**Pull request by Draeggon**
a little shitty commit that will help me to write unit test

### CODE REVIEW

File: doc/tools/yamllint.md [hunk 1]
```diff
@@ -8,12 +8,12 @@
 
  Run yamllint in a file
 ```sh
-yamllint path_to_file
+yamllint -c path/to/config-yamllint.yaml/in/server path_to_file
```
Comment (Benvii) [inline doc/tools/yamllint.md:11]: Why don't you put relative path to the config file you have commited ?

`yamllint -c ../../server/scratchy_server/config-yamllint.yaml path_to_file`

**Review by Benvii — COMMENTED** [review]
Config file tested it works. Maybe juste make small update in the documentation to be more concrete.

File: doc/tools/yamllint.md (hunk 1 above)
Comment (Draeggon) [reply doc/tools/yamllint.md:11]: cause you could be in any directory when you call yamllint

Comment (Benvii) [reply doc/tools/yamllint.md:11]: Well at least mention the path with full location `server/scratchy_server/config-yamllint.yaml` might be more understandable the `stuff/in/server` is a bit confusing

**Review by Benvii — APPROVED** [review]
Thank @Draeggon :+1:

File: doc/tools/yamllint.md (hunk 1 above)
Comment (Benvii) [reply doc/tools/yamllint.md]: Done in 
94e05e3

**Pull request merged**